shanoir-ng-front/src/app/examinations/shared/examination.service.ts

```diff
@@ -25,6 +25,7 @@ import { ExaminationDTO, ExaminationDTOService } from './examination.dto';
 import { ServiceLocator } from '../../utils/locator.service';
 import { LoadingBarComponent } from '../../shared/components/loading-bar/loading-bar.component';
 import { Subscription } from 'rxjs'
+import {saveAs} from "file-saver";
```

Comment:
What is it for ? I might be wrong but it seems that those 2 ts have been unintentionally commited.